lib/airvantage.js

```diff
@@ -211,6 +211,24 @@ _.each(SYSTEM_LIFECYCLE_OPERATIONS, function(operationName) {
     };
 });
 
+AirVantage.prototype.queryMultiRawDataPoints = function(params) {
+    console.log("queryMultiRawDataPoints with params " + JSON.stringify(params));
+    var url = this.serverUrl + "/" + BASE_API_URL + "/systems/data/raw";
+    var query = {
+        query: this._buildQuery(params),
+        headers: {
+            "Content-type": "application/json"
+        },
+        json: true
+    };
+
+    return got(url, query)
+        .then(function(res) {
+            return res.body;
+        }).catch(function(e) {
```

Comment:
Errors should not be catched unless we want to process them or hide them.
I know it's done like that in all the file I've cleaned it in another branch :smile: